elasticsearch/src/main/scala/akka/stream/alpakka/elasticsearch/scaladsl/ElasticsearchFlow.scala

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (C) 2016-2017 Lightbend Inc. <http://www.lightbend.com>
+ */
+package akka.stream.alpakka.elasticsearch.scaladsl
+
+import akka.NotUsed
+import akka.stream.alpakka.elasticsearch.{ElasticsearchFlowStage, ElasticsearchSinkSettings, IncomingMessage}
+import akka.stream.scaladsl.Flow
+import org.elasticsearch.client.{Response, RestClient}
+import spray.json.{DefaultJsonProtocol, JsObject, JsonWriter}
+
+object ElasticsearchFlow {
+
+  /**
+   * Scala API: creates a [[ElasticsearchFlowStage]] that accepts as JsObject
+   */
+  def apply(indexName: String, typeName: String, settings: ElasticsearchSinkSettings)(
+      implicit client: RestClient
+  ): Flow[IncomingMessage[JsObject], Response, NotUsed] =
+    Flow
+      .fromGraph(
+        new ElasticsearchFlowStage(indexName, typeName, client, settings)(DefaultJsonProtocol.RootJsObjectFormat)
+      )
+      .mapAsync(settings.parallelism)(identity)
```

Comment:
The parallelism setting has no effect as it doesn't matter how many times you call `identity` concurrently. Just use `1` instead.